### PR TITLE
Fixing errors with custom registration form 

### PR DESCRIPTION
**Pull request by wbloszyk**
Fix error with difrence user form (define by user in sonata_user config and define in sonata.user.registration.form service). Also fix null $url variable.

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Please make a clearer PR title and commit message.

**Comment by wbloszyk** [comment]
@Soullivaneuh 
Is it better now ?

**Comment by Soullivaneuh** [comment]
Yes. Now you could do it on the commit too. :-)

BTW, some explanations of this PR with some samples would be useful. Can you handle it?

**Comment by wbloszyk** [comment]
I think this issue tell everything about this RP.

https://github.com/sonata-project/ecommerce/issues/345

**Comment by Soullivaneuh** [comment]
Thanks for the link.

Something to do before start review:

 * Rename the commit message 
 * Add related tests
 * Fix CS: https://travis-ci.org/sonata-project/SonataUserBundle/jobs/69108434#L648

Could you please look at this? Thanks.

File: Controller/RegistrationFOSUser1Controller.php [hunk 1]
```diff
@@ -67,6 +68,10 @@ public function registerAction()
 
             $this->setFlash('fos_user_success', 'registration.flash.user_created');
 
+            if($url === null){
```
Comment (Soullivaneuh) [inline Controller/RegistrationFOSUser1Controller.php]: Spaces and Yoda comparison are needed.

**Comment by core23** [comment]
:+1:

**Comment by core23** [comment]
@wbloszyk Are you still working on this pr?

**Comment by core23** [comment]
Ping @wbloszyk

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**